openquake/server/tests/db/upgrade_manager_test.py

```diff
@@ -55,20 +55,25 @@ class UpgradeManagerTestCase(unittest.TestCase):
     # automatically created an destroyed in the setUp/tearDown methods.
     # This test consider the 5 scripts in openquake.server.tests.db.upgrades:
 
-    ## 0000-base_schema.sql
-    ## 0001-populate.sql
-    ## 0002-a-syntax-error.sql
-    ## 0003-insert-error.sql
-    ## 0005-populate_model.py
+    # 0000-base_schema.sql
+    # 0001-populate.sql
+    # 0002-a-syntax-error.sql
+    # 0003-insert-error.sql
+    # 0005-populate_model.py
 
     def setUp(self):
-        global conn
-        os.environ.setdefault("DJANGO_SETTINGS_MODULE",
-                              "openquake.server.settings")
-        conn = connection.cursor().connection  # psycopg2 connection
-        conn.autocommit = False
-        conn.cursor().execute('CREATE SCHEMA test')
-        conn.commit()  # make sure the schema really exists
+        global conn, tmpfile
+        tmpfile = tempfile.NamedTemporaryFile()
+        tmpfile.close()
+        conn = sqlite3.connect(tmpfile.name)
+        tables = conn.execute(
+            "SELECT name FROM sqlite_master WHERE name LIKE 'test_%'")
```

Comment:
Since we are using separated DBs to run the test, the 'production' DB should be always empty at the beginning.